src/System.IO.FileSystem/tests/Directory/GetFiles_str_Str.cs

```diff
@@ -11,454 +12,274 @@
-            strLoc = "Loc_98yg5";
+            //suffix only should return the empty array
+            Assert.Empty(GetFiles(Path.Combine(TestDirectory, testDir1Str), testFile1.Substring(3)));
+        }
+    }
 
-            iCountTestcases++;
-            try
-            {
-                Directory.GetFiles(",", "*");
-                iCountErrors++;
-                printerr("Error_2y675! Expected exception not thrown");
-            }
-            catch (DirectoryNotFoundException)
-            {
-            }
-            catch (Exception exc)
-            {
-                iCountErrors++;
-                printerr("Error_249y6! Incorrect exception thrown, exc==" + exc.ToString());
-            }
+    [Fact]
+    public void SearchPatternSuperset()
+    {
+        String testDir1Str = GetTestFileName();
+        String testDir11Str = GetTestFileName();
+        DirectoryInfo testDir = new DirectoryInfo(TestDirectory);
+        DirectoryInfo testDir1 = testDir.CreateSubdirectory(testDir1Str);
+        testDir1.CreateSubdirectory(testDir11Str);
+
+        String testFile1 = GetTestFileName();
+        String testFile2 = "ca1" + testFile1;
+        testFile1 = "ca2" + testFile1;
+        String testFile3 = GetTestFileName();
+        String searchPattern = String.Format("???{0}", testFile1.Substring(3));
+
+        using (new FileInfo(Path.Combine(TestDirectory, testDir1Str, testDir11Str, GetTestFileName())).Create())
+        using (new FileInfo(Path.Combine(TestDirectory, testDir1Str, testFile1)).Create())
+        using (new FileInfo(Path.Combine(TestDirectory, testDir1Str, testFile2)).Create())
+        using (new FileInfo(Path.Combine(TestDirectory, testDir1Str, testFile3)).Create())
+        {
+            String[] results = GetFiles(Path.Combine(TestDirectory, testDir1Str), searchPattern);
+            Assert.Equal(2, results.Length);
+            Assert.Contains(Path.Combine(TestDirectory, testDir1Str, testFile1), results);
+            Assert.Contains(Path.Combine(TestDirectory, testDir1Str, testFile2), results);
 
-            strLoc = "Loc_98001";
+            //suffix only should return the empty array
+            Assert.Empty(GetFiles(Path.Combine(TestDirectory, testDir1Str), testFile1.Substring(3)));
+        }
+    }
 
-            iCountTestcases++;
-            try
-            {
-                Directory.GetFiles("DoesNotExist", "*");
-                iCountErrors++;
-                printerr("Error_2y76b! Expected exception not thrown");
-            }
-            catch (DirectoryNotFoundException)
-            {
-            }
-            catch (Exception exc)
-            {
-                iCountErrors++;
-                printerr("Err_24y7g! Incorrect exception thrown, exc==" + exc.ToString());
-            }
+    [Fact]
+    [ActiveIssue(846, PlatformID.AnyUnix)]
+    public void CharacterTests()
+    {
+        // TODO: [ActiveIssue(846)] Globalization on Unix
```

Comment:
Now that we actually have the ActiveIssue on this, the comment for the same thing can be removed.